acvm_js/test/node/execute_circuit.test.ts

```diff
@@ -205,3 +205,29 @@ it("successfully executes two circuits with same backend", async function () {
   expect(solvedWitness0).to.be.deep.eq(expectedWitnessMap);
   expect(solvedWitness1).to.be.deep.eq(expectedWitnessMap);
 });
+
+it.only("successfully executes 500 circuits with same backend", async function () {
```

Comment:
You probably don't want the `only` in here?

Reply:
Good point, just fixed